assembly/configuration/src/main/resources/workflow.xml

```diff
@@ -37,6 +37,9 @@
         <step>
             <name>License Knowledgebase Resolver</name>
             <classHint>org.eclipse.sw360.antenna.workflow.processors.LicenseKnowledgeBaseResolver</classHint>
+            <configuration>
```

Comment:
I do not think, the standard workflow should state a configuration here. Instead the standard priorities should be followed.